core/services/job/job_orm_test.go

```diff
@@ -150,19 +150,26 @@ func TestORM(t *testing.T) {
 		require.NoError(t, orm.RecordError(jobSpec.ID, ocrSpecError1))
 		require.NoError(t, orm.RecordError(jobSpec.ID, ocrSpecError2))
 
-		expected1ID := specErrors[1].ID + 1
-		expected2ID := specErrors[1].ID + 2
-		specErr1, err := orm.FindSpecError(expected1ID)
+		var updatedSpecError []job.SpecError
+
+		err = db.Select(&updatedSpecError, "SELECT * FROM job_spec_errors ORDER BY id ASC")
+		require.NoError(t, err)
+		require.Len(t, updatedSpecError, 4)
```

Comment:
Note: This problem existed here earlier, not introduced by your PR. Just pointing out since I noticed it now.

So here we seem to have an implicit assumption that this test always runs after the previous test, which had inserted 2 errors into the job_spec_errors table.
Problem with such assumptions is that later if someone refactors and moves tests around, or have a filter which only runs this test, instead of above one, then this will start failing.

I would say, a safer approach may be to clear this table at the start of this test. So it ensures previous state from previous test doesn't affect this test. Makes all tests idempotent.